firmware/octopi_firmware_v2/main_controller_teensy41/main_controller_teensy41.ino

```diff
@@ -274,7 +293,7 @@ typedef struct pid_arguments {
 PID_ARGUMENTS axis_pid_arg[N_MOTOR];
 
 // home safety margin
-uint16_t home_safety_margin[4] = {4, 4, 4, 4};
+uint16_t home_safety_margin[N_MOTOR] = {4, 4, 4, 4};
```

Comment:
@veerwang need to change it back to `uint16_t home_safety_margin[4] = {4, 4, 4, 4};`

Reply:
In the future, suppose there are over 4 drivers slot in the device, only change N_MOTOR to new value.

Reply:
OK, I would change the N_MOTOR to 4